frontend/javascripts/router.js

```diff
@@ -365,7 +364,6 @@ class ReactRouter extends React.Component<Props> {
                 component={ScriptListView}
                 exact
               />
-              <Route path="/help/keyboardshortcuts" component={KeyboardShortcutView} />
```

Comment:
How about redirecting to `https://docs.webknossos.org/reference/keyboard_shortcuts`?

Reply:
hmm. why do you think that is necessary?

Reply:
If an outdated wk version is shown (due to old cache or long-running tab) or the help link is used externally (e.g., bookmarks or in some lab wikis/tutorials), the old link will point to 404. Not a major problem, but easy to mitigate, I guess.